pkg/typesystem/typesystem.go

```diff
@@ -0,0 +1,31 @@
+package typesystem
+
+type SchemaVersion int
+
+const (
+	SchemaVersionUnspecified SchemaVersion = iota
+	SchemaVersion1_0
+	SchemaVersion1_1
+)
+
+func NewSchemaVersion(s string) SchemaVersion {
```

Comment:
Please add a `typesystem_test.go` test file and test this and the `String()` method below.

Reply:
Tests added. Thanks!